lua/nvim-tree/lib.lua

```diff
@@ -81,6 +81,16 @@ function M.get_last_group_node(node)
   return next
 end
 
+function M.get_all_nodes_in_group(node)
+  local next = utils.get_parent_of_group(node)
```

Comment:
Please don't overload the builtin function `next`.

I know it's been used elsewhene...